bundles/specmate-connectors/src/com/specmate/connectors/internal/ConnectorTask.java

```diff
@@ -63,16 +65,23 @@ private void syncRequirementsFromSources() {
 					greatestUnhandledIndex = upperIndexExclusive;
 					List<Requirement> tosync = Arrays.asList(current);
 					syncContainers(localContainer, tosync, source);
-					transaction.commit();
+
+					transaction.doAndCommit(new IChange<Object>() {
+						@Override
+						public Object doChange() throws SpecmateException, SpecmateValidationException {
+							return null;

```

Comment:
that seems strange, there is nothing happening inside the callback method. Probably the call to syncContainers should go there

Reply:
Fixed, as well as all other instances.